packages/ethash/src/index.ts

```diff
@@ -0,0 +1,241 @@
+import {
+  BN,
+  keccak,
+  keccak256,
+  rlphash,
+  zeros,
+  bufferToInt,
+  TWO_POW256
+} from 'ethereumjs-util'
+import {
+  params,
+  fnv,
+  fnvBuffer,
+  bufReverse,
+  getEpoc,
+  getCacheSize,
+  getFullSize,
+  getSeed
+} from './util'
+import type { LevelUp } from 'levelup'
+const async = require('async')
+const xor = require('buffer-xor')
+
+// ethereumjs-block v3 will have types
+type Block = any
+type BlockHeader = any
```

Comment:
todo: can probably remove these and import the types from block package

Reply:
resolved in https://github.com/ethereumjs/ethereumjs-vm/pull/766/commits/213cd058ba51d906a16007935e72270cf5fc62a9